src/widget/wtracktableview.cpp

```diff
@@ -110,6 +111,11 @@ WTrackTableView::WTrackTableView(QWidget * parent,
 
     connect(this, SIGNAL(scrollValueChanged(int)),
             this, SLOT(slotScrollValueChanged(int)));
+
```

Comment:
I don't quite understand this idiom, it looks like it leaks the shortcut object since it's never deleted.  Or is that ok, because it doesn't need to be?  (I understand you're just cut and pasting the original code)